From c8bb827526aabe9978428cffaa0b3b73ef83f9d9 Mon Sep 17 00:00:00 2001
From: Artem Radchenko <artem.radchenko@globallogic.com>
Date: Mon, 22 Jun 2020 13:25:38 +0300
Subject: [PATCH 11/12] Fix addUser button is not clickable on lockscreen

Reenable addUser button when user is switched to prevent button is not
clickable case when returning to user switcher from lockscreen.

Test: Manual
JiraId: ADM-4897
Signed-off-by: Artem Radchenko <artem.radchenko@globallogic.com>
Change-Id: I60f0d02a9898921bd4d1d0869b6126f19ee1ff71
---
 .../android/systemui/statusbar/car/UserGridRecyclerView.java  | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/packages/CarSystemUI/src/com/android/systemui/statusbar/car/UserGridRecyclerView.java b/packages/CarSystemUI/src/com/android/systemui/statusbar/car/UserGridRecyclerView.java
index 827a59eddf5..17f1c81ea30 100644
--- a/packages/CarSystemUI/src/com/android/systemui/statusbar/car/UserGridRecyclerView.java
+++ b/packages/CarSystemUI/src/com/android/systemui/statusbar/car/UserGridRecyclerView.java
@@ -347,6 +347,10 @@ public class UserGridRecyclerView extends RecyclerView implements
                 if (user != null) {
                     mCarUserManagerHelper.switchToUser(user);
                 }
+                // Reenable the add button when user is switched
+                if (mAddUserView != null) {
+                    mAddUserView.setEnabled(true);
+                }
             }
         }
 
-- 
2.28.0

